packages/aws-cdk/test/api/cloudformation-deployments.test.ts

```diff
@@ -142,3 +163,700 @@ test('if toolkit stack cannot be found but SSM parameter name is present deploym
 
   expect(requestedParameterName!).toEqual('/some/parameter');
 });
+
+test('readCurrentTemplateWithNestedStacks() can handle non-Resources in the template', async () => {
+  const cfnStack = new FakeCloudformationStack({
+    stackName: 'ParentStackWithOutputAndParameter',
+    stackId: 'StackId',
+  });
+  CloudFormationStack.lookup = (async (_, stackName: string) => {
+    switch (stackName) {
+      case 'ParentStackWithOutputAndParameter':
```

Comment:
Where's the Output and Parameter here...? I see it in the Nested Stack, but not in the parent Stack...?

Reply:
the Output / Parameter are supposed to only be in the nested stack. The parent stack's name should reflect that, maybe `ParentOfStackWithOutputAndParameter`?

Reply:
Yep, definitely better.